plugins/action/ci_script.py

```diff
@@ -186,16 +187,17 @@ def run(self, tmp=None, task_vars=None):
             file_task.args.pop(arg_name, None)
 
         fnum = len(glob.glob(f"{output_dir}/ci_script_*"))
-        t_name = re.sub(r"([^\x00-\x7F]|\s)+", "_", self._task.name).lower()
+        t_name = re.sub(r"([^\x00-\x7F]|\s|-)+", "_", self._task.name).lower()
         chdir_path = task_args.get("chdir", None)
+        log_path = output_dir.parent.joinpath(
+            "logs", f"ci_script_{fnum:03}_{t_name}.log"
```

Comment:
for the sake of consistency, maybe we should ensure names aren't too long? A truncate() (or equivalent) to 20 chars or so (like the run_hook role) ?